task 1/shell code.txt

```diff
@@ -0,0 +1,6 @@
+chengyi@chengyi-G3-3579:~$ git clone https://github.com/facebook/folly.git
+chengyi@chengyi-G3-3579:~$ cd folly
+chengyi@chengyi-G3-3579:~/folly$ mkdir _build
+chengyi@chengyi-G3-3579:~/folly$ cd build
```

Comment:
You made a `_build` but you get into `build`. Why?

Reply:
That's a silly mistake.Thanks for your correction.